settings.py

```diff
@@ -15,6 +15,8 @@
     }
 }
 
-USE_TZ = False
+DEBUG = True
```

Comment:
The `DEBUG` setting is set to `True`. Remember to set this to `False` in a production environment to avoid exposing sensitive information.